bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -197,7 +214,9 @@ if args.make_signal_kde:
     logging.info('Evaluating signal KDE')
     signal_kde = kde_awkde(signal_samples, temp_samples, \
                            alp=optalpha, gl_bandwidth=optbw)
-    f_dest.create_dataset("data_kde", data=signal_kde)
+    modified_kde = new_signal_kde(signal_kde, frac=args.extra_cpt_fraction, \
```

Comment:
what if the user doesn't give the extra cpt fraction and volume arguments?  should the code still be able to run?